Initial Draft for SAI_1_9_1 Release notes

kannankvs opened the pull request:
Initial Draft for SAI_1_9_1 Release notes. Kindly review

rlhui commented:
@j-bos , @dipankar-ba , @ashutosh-agrawal , @SidharajU, @VenkatCisco , @erohsik, @mikeberesford , @msomasundaran, @kcudnik , @marian-pritsak , @JaiOCP  - would you please help review this release notes for the last SAI release? Thanks.

dipankar-ba commented on doc/SAI_1.9.1 Release notes.md:
```diff
@@ -0,0 +1,651 @@
+
+### Adding IPsec support
+
+A new API saiipsec.h has been added part of this current release to add an attribute for supporting both per SA and GLOBAL counters. Globals are aggregates for ingress and egress direction across all SAs. Per SA stats are directional per SA.
```
This description is incorrect.  It should be replaced with something like:
A new IPsec API saiipsec.h has been added part of this current release.  For more detail, please refer to https://github.com/opencomputeproject/SAI/doc/IPsec/SAI_IPsec_API_proposal.md

kannankvs replied:
addressed

rlhui replied:
@dipankar-ba  - please check, thx

kcudnik commented:
why there is so many commits suddenly here?

kannankvs commented:
Hi,
While addressing the comments, we ended up in a wrong "rebasing" which resulted in multiple commits.
We reverted back the commits.

Regards,
Kannan.KVS

From: Kamil Cudnik ***@***.***>
Sent: Friday, February 25, 2022 6:00 PM
To: opencomputeproject/SAI
Cc: KVS, Kannan; Author
Subject: Re: [opencomputeproject/SAI] Initial Draft for SAI_1_9_1 Release notes (PR #1397)


[EXTERNAL EMAIL]

why there is so many commits suddenly here?

-
Reply to this email directly, view it on GitHub [github.com]<https://urldefense.com/v3/__https:/github.com/opencomputeproject/SAI/pull/1397*issuecomment-1050813048__;Iw!!LpKI!wAT4V-adAVfIiQiYPBt7-MXqq8l01Q96cF5LsXQPSXCBLsB7GvPvP986siQXjFY$>, or unsubscribe [github.com]<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ABX64IB4UUKA62JK5J6YSPDU45Y5FANCNFSM5L2Z5AAQ__;!!LpKI!wAT4V-adAVfIiQiYPBt7-MXqq8l01Q96cF5LsXQPSXCBLsB7GvPvP9861OKNTIk$>.
Triage notifications on the go with GitHub Mobile for iOS [apps.apple.com]<https://urldefense.com/v3/__https:/apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675__;!!LpKI!wAT4V-adAVfIiQiYPBt7-MXqq8l01Q96cF5LsXQPSXCBLsB7GvPvP986U9sx7IQ$> or Android [play.google.com]<https://urldefense.com/v3/__https:/play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign*3Dnotification-email*26utm_medium*3Demail*26utm_source*3Dgithub__;JSUlJSU!!LpKI!wAT4V-adAVfIiQiYPBt7-MXqq8l01Q96cF5LsXQPSXCBLsB7GvPvP986SVKzj9M$>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>


Internal Use - Confidential
